node/tests/SharedTests.ts

```diff
@@ -2054,6 +2055,47 @@ export function runBaseTests<Context>(config: {
         config.timeout,
     );
 
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+        `test blpop test_%p`,
+        async (protocol) => {
+            await runTest(async (client: BaseClient) => {
+                expect(
+                    await client.rpush("blpop-test", ["foo", "bar", "baz"]),
+                ).toEqual(3);
+                // Test basic usage
+                expect(await client.blpop(["blpop-test"], 0.1)).toEqual([
+                    "blpop-test",
+                    "foo",
+                ]);
+                // Delete all values from list
+                expect(await client.del(["blpop-test"])).toEqual(1);
+                // Test null return when key doesn't exist
+                expect(await client.blpop(["blpop-test"], 0.1)).toEqual(null);
+
+                try {
+                    await client.set("foo", "bar"),
+                        expect(await client.blpop(["foo"], 0.1)).toThrow();
+                } catch (e) {
+                    expect(e as RequestError);
+                }
+
```

Comment:
Is it possible  to test in node `blpop` with 0 timeout?
see https://github.com/Bit-Quill/glide-for-redis/pull/196/files#diff-1c3a6923c1bf3dd19a1c98c6523f881876a156a8316c3b3f5dc6801a405382ceR882-R883 and https://github.com/Bit-Quill/glide-for-redis/pull/209/files#diff-5db9a3bd41d647cfdba521ea248ea07d7c78bc9aaa41a26bb923d4f92a3efdd2R1694-R1698